src/Models/Payment.php

```diff
@@ -13,8 +13,6 @@ class Payment extends BaseModel
     protected $casts = [
     ];
 
-    protected $refund = app('refund');
```

Comment:
This is not legal PHP - can't use a function in member assignment.  I'm updating the checkout packages to require their true dependencies and stumbled on this trying to run tests.